modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -372,11 +374,11 @@ CREATE OR REPLACE TRIGGER SP_CLAIM_MAPPING_TRIG
 ALTER TABLE SP_CLAIM_MAPPING ADD CONSTRAINT CLAIMID_APPID_CONSTRAINT FOREIGN KEY (APP_ID) REFERENCES SP_APP (ID) ON DELETE CASCADE
 /
 CREATE TABLE SP_ROLE_MAPPING (
-            ID INTEGER,
-            TENANT_ID INTEGER NOT NULL,
-            IDP_ROLE VARCHAR (255) NOT NULL ,
+	    	ID INTEGER,
+	    	TENANT_ID INTEGER NOT NULL,
+	    	IDP_ROLE VARCHAR (255) NOT NULL ,
             SP_ROLE VARCHAR (255) NOT NULL ,
-            APP_ID INTEGER NOT NULL,
+	    	APP_ID INTEGER NOT NULL,
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04